plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/catalog/hms/TrinoHiveCatalog.java

```diff
@@ -319,6 +320,17 @@ public List<SchemaTableName> listTables(ConnectorSession session, Optional<Strin
         return tablesListBuilder.build().asList();
     }
 
+    @Override
+    public Optional<Iterator<RelationColumnsMetadata>> streamRelationColumns(
+            ConnectorSession session,
+            Optional<String> namespace,
+            UnaryOperator<Set<SchemaTableName>> relationFilter,
+            Predicate<SchemaTableName> isRedirected)
+    {
+        // TODO
```

Comment:
Can we remove or update this comment describing the next step?

Reply:
will remove